tools/topology/topology2/include/pipelines/cavs/dai-copier-eqiir-module-copier-capture.conf

```diff
@@ -86,6 +86,7 @@ Class.Pipeline."dai-copier-eqiir-module-copier-capture" {
 		copier."2" {
 			type buffer
 			copier_type	"module"
+			node_type	$BUFFER_COPIER
```

Comment:
that pipeline should be broken-up in two: io-gateway and processing pipelines.

Reply:
yes i want to do that for both the host and dai sides. coming soon